src/lib/_navigation.scss

```diff
@@ -26,6 +26,10 @@ $navigationBtnWidth: 50px;
     &:hover {
         opacity: 1;
     }
+
+    &:focus {
+        opacity: 1;
```

Comment:
Can you do &:hover, &:focus and combine the two?

Reply:
👍